doc/reverse_engineering/civilizations.md

```diff
@@ -0,0 +1,100 @@
+# Civilization attributes
+
+## Age of Empires I
+
+### Civilization specials
+
+Civilization | Bonus
```

Comment:
You need an extra line of `-|-` between the header row and the content. Otherwise it will not be recognized as a table in Markdown.

```
Civilization | Bonus
-------------|------
Assyrian | +40% Archery Range unit fire rate; Villagers 30% faster.
Babylonian | Double wall and tower hit points; +30% Priest rejuvenation rate; +30% stone mining.
Choson | +80 Long Swardsman and Legion hit points, +2 tower range, -30% Priest cost
```
looks like this

Civilization | Bonus
------|------
Assyrian | +40% Archery Range unit fire rate; Villagers 30% faster.
Babylonian | Double wall and tower hit points; +30% Priest rejuvenation rate; +30% stone mining.
Choson | +80 Long Swardsman and Legion hit points, +2 tower range, -30% Priest cost